### ISSUE TITLE
Life Expectancy map not displaying

### ISSUE THREAD

**Issue by knweaver** [smartchicago/chicago-atlas#396]
I think when Life expectancy was corrected for #362 it caused the map to no longer display

![image](https://user-images.githubusercontent.com/39635248/54207098-92861a00-44a7-11e9-9f84-a3e91d7e2da1.png)

**Comment by kfranzen24** [comment]
[Life_Expectancy.xlsx](https://github.com/smartchicago/chicago-atlas/files/3059223/Life_Expectancy.xlsx)

The maps are not displaying because there are no data in any of the Rate columns. (see attached)

**Comment by knweaver** [comment]
Correct. It is not a rate, it is a number. The number column should display in the map (as specified in the description template).
________________________________
